test/e2e/main/main_test.go

```diff
@@ -1450,3 +1450,61 @@ func createCacheWithCR(cacheName string, nameSpace string, clusterName string) *
+	hostAddr, client := tutils.HTTPClientAndHost(ispn, testKube)
+	createCacheWithXMLTemplate(cacheName, hostAddr, template, client)
+
+	//Generate tons of random entries
+	for key := 1; key < 60000; key++ {
```

Comment:
Would be nice to stop the cycle when OOM hits so it doesn't load bunch of other entries

Reply:
With the new code, when it hits an OOM it breaks the loop instead of put all the occurences